src/main/kotlin/com/atlassian/performance/tools/jiraactions/jql/JqlImpl.kt

```diff
@@ -0,0 +1,87 @@
+package com.atlassian.performance.tools.jiraactions.jql
+
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.jql.Jql
+import com.atlassian.performance.tools.jiraactions.api.jql.JqlContext
+import com.atlassian.performance.tools.jiraactions.api.jql.JqlSupplier
+import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
+
+internal class JqlImpl(private val query: String,
```

Comment:
"JQL implementation". What makes this one different from the other implementations?
If none other exist, how *could* they differ?

It seems that the interface allows dynamic `query` and `supplier` logic, but this impl simply reuses the provided fields. Suggestions: `StaticJql`, `SuppliedJql`.